database/src/main/kotlin/no/nav/su/se/bakover/database/JournalføringOgBrevdistribusjonMapper.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.su.se.bakover.database
+
+import no.nav.su.se.bakover.domain.brev.BrevbestillingId
+import no.nav.su.se.bakover.domain.eksterneiverksettingssteg.JournalføringOgBrevdistribusjon
+import no.nav.su.se.bakover.domain.journal.JournalpostId
+
+internal object JournalføringOgBrevdistribusjonMapper {
+    fun idToObject(
+        iverksattJournalpostId: JournalpostId?,
+        iverksattBrevbestillingId: BrevbestillingId?
+    ) = when {
```

Comment:
Synes det er greit å spesifisere en returtype for så store blokker. Det hjelper ikke at funksjonen heter toObject :]

Reply:
Refaktorerer og bytter ut mapperen med companion.